src/EoS/Nuclear/CPU_EoS_Nuclear.cpp

```diff
@@ -236,12 +241,21 @@ static real EoS_DensEint2Pres_Nuclear( const real Dens_Code, const real Eint_Cod
       printf( "        Passive scalars:" );
       for (int v=0; v<NCOMP_PASSIVE; v++)    printf( " %d=%13.7e", v, Passive_Code[v] );
       printf( "\n" );
-      printf( "        EoS error code: %d\n", Err );
 #     endif
+      printf( "        EoS error code: %d\n", Err );
    }
 #  endif // GAMER_DEBUG
 
 
+// store extra output variables if required
+   /*
+   if ( ExtraInOut != NULL )
+   {
+      ExtraInOut[...] = ...;
```

Comment:
The ExtraInOut[] is probably TestProblem depends, is there a better place to customize it? The current form is fine.

Reply:
Currently it cannot be customized by individual test problems.

For the nuclear EoS, we can simply store all useful output fields since `nuc_eos_C_short()` will always return them anyway (so it won't hurt performance).